docs/changelog.md

```diff
@@ -124,23 +134,33 @@ description: |
   - Improved layout of nested attachments resulting from code like
     `[#let a0 = $a_0$; $a0^1$]`
   - Improved layout of primes close to superscripts
-  - Typst now makes use of math-specific height-dependant kerning information in
+  - Typst now makes use of math-specific height-dependent kerning information in
     some fonts for better attachment layout
-  - The `floor` and `ceil` functions in math are now callable symbols instead,
+  - The `floor` and `ceil` functions in math are now callable symbols instead <of what>,
     such that `[$ floor(x) = lr(floor.l x floor.r) $]`
   - The [`mat.delim`]($math.mat.delim), [`vec.delim`]($math.vec.delim), and
     [`cases.delim`]($math.cases.delim) parameters now allow any character that
-    is considered a delimiter or "fence" (e.g. |) by Unicode. The
+    is considered a delimiter or "fence" (e.g., |) by Unicode. The
     `{delim: "||"}` notation is _not_ supported anymore and should be replaced
     by `{delim: bar.double}` (**Minor breaking change**)
   - Added [`vec.align`]($math.vec.align) and [`mat.align`]($math.mat.align)
     parameters
   - Added [`underparen`]($math.underparen), [`overparen`]($math.overparen),
     [`undershell`]($math.undershell), and [`overshell`]($math.underparen)
-  - Added `~` shorthand for `tilde.op` (**Minor breaking change**)
-  - Fixed baseline alignment of equation numbers
+  - Added `~` shorthand for `tilde.op`. See
+    [#4706](https://github.com/typst/typst/pull/4706) for details. (**Minor
+    breaking change**)
+    <!-- Maybe add somewhere how you can get the previous output? -->
+    <!-- $a ~ b$ -->
+    <!-- $a "~" b$ -->
+    <!-- $a class("normal", ~) b$ -->
+    <!-- $a class("normal", "~") b$ -->
+    <!-- $a class("binary", "~") b$ -->
+  - Fixed baseline alignment of equation numbers. See
```

Comment:
I dropped these links because I don't generally link the issues. It adds too much noise in my opinion.